src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -871,9 +897,7 @@ export default class FilingHistoryList extends Mixins(
       item.availableOnPaperOnly ||
       item.isTypeStaff ||
       item.isFutureEffective ||
-      this.isStatusCorrected(item) ||
       this.isTypeAlteration(item) ||
-      this.isTypeCorrection(item) ||
```

Comment:
I have not heard that we are allowing corrections to corrections or already-corrected filings, so please revert this code.